career-growth.md

```diff
@@ -0,0 +1,69 @@
+# Conscious Career Growth
+
+It used to be the case that people would stay with one employer for their entire careers. This is no longer the case today. It is normal and expected for most people to move from role to role throughout their lives. That said, you are at Packback right now for a reason! Your job with Packback is an alliance that benefits both parties:
+
+* You apply your unique skills, vision and expertise to move Packback forward as a company.
+* Your work at Packback pushes you to learn new skills, develop your vision and build new expertise.
+
+This exercise is designed to help you think through your ideal future. We'll be looking at the intersection of how your skills, vision and expertise move Packback forward, and how your growth in turn will help Packback to accelerate towards its mission.
+
+## How This Works
+
+First, this is **not a performance conversation, there is no way to fail at this**. We already have tools in place to help you get to your next promotion (performance reviews every trimester, peer reviews yearly, a regular cadence of opportunities for promotion, (1-on-1s)[1-on-1s.md] for weekly coaching). This exercise is not about your next promotion. Conscious career growth is about longer timescales, so try and forget the next year for now.
+
+**This is not an exercise with an end date**. There is no "up and out." When you reach one career milestone it is simply time to look forward to the next. You'll be exploring the overlap between your aspirations and your career at Packback, and revisiting and tweaking regularly. Going into this meeting, both parties should be in a **directional, aspirational mindset!** Don't be afraid or embarrassed to think big. 
+
+This exercise is done with two meetings:
+
+* A skip-level meeting with your manager's manager where you fill out the three questions below
```

Comment:
Since there's a period at the end of the second bullet point, should there be one here too?